tests/test_basic_ops_dict.py

```diff
@@ -255,10 +255,13 @@ def foo(a: int, b: int) -> int:
     assert 12 == foo(**c)
 
 
-def test_items() -> None:
+def test_items_iterator_behavior() -> None:
     c = OmegaConf.create({"a": 2, "b": 10})
-    assert sorted([("a", 2), ("b", 10)]) == sorted(list(c.items()))
+    assert list(c.items()) == [("a", 2), ("b", 10)]
```

Comment:
I wouldn't worry about it myself, but FYI the deterministic ordering of Python dictionaries is considered an implementation detail of CPython in 3.6.

Reply:
We have CI coverage so I am not worried.